buildroot-external/patches/occu/0154-WebUI-DynamicResizing.patch

```diff
@@ -113,6 +113,138 @@
        gotoLoginPage = function() {
          location.href = "/login.htm";
        };
+--- occu/WebUI/www/rega/pages/tabs/control/devices.htm.orig
++++ occu/WebUI/www/rega/pages/tabs/control/devices.htm
+@@ -13,14 +13,9 @@
+   arItems = new Array();
+ </script>
+ 
+-<table width="100%" class="CLASS03900" cellpadding="0" cellspacing="0">
+-  <colgroup>
+-    <col style="width:20%;"/>
+-    <col style="width:80%;"/>
+-  </colgroup>
+-  <tr>
++<div id="devicepage" class="CLASS03900" style="width:100%">

```

Comment:
How about moving that `style="width:100%"` to the global CSS? IMHO we should try to avoid using element-based style statements in future and prefer to move all css statements to `style.css`

Reply:
Sure. I was also preparing for this already by introducing the id `devicepage` for the `<div>`.